client/components/domains/register-domain-step/index.jsx

```diff
@@ -687,7 +691,7 @@ class RegisterDomainStep extends React.Component {
 	};
 
 	handleSubdomainSuggestions = ( domain, timestamp ) => subdomainSuggestions => {
-		this.props.onDomainsAvailabilityChange( true );
+		this.props.onDomainsAvailabilityChange( true, 0 );
```

Comment:
Why `0` explicitly? What does it mean in this context? I think it'd be more straightforward to just set that parameter to default to `0` and not pass it here (and in other places). Bonus: less modificed lines in this patch ;)

Reply:
Yeah. Good point. It is hard to figure out why that `0` is there.